Examples/CustomLoginPageWithHtmlHelpers/Pages/_ViewImports.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@using InMemory
```

Comment:
There should be a check over the whole solution to find any bad namespaces that are not right for the project